### PR TITLE
removed dependency on text-show

### PR DESCRIPTION
**Pull request by pjljvandelaar**
On windows, TorXakis can not use integer-gmp due to license issues.
text-show uses integer-gmp, so text-show can not be used.

Also removed a number of warnings (found with -Wall / --pedantic)

### CODE REVIEW

File: sys/behave/behave.cabal [hunk 1]
```diff
@@ -67,10 +67,11 @@ test-suite behave-test
   build-depends:       base
                      , containers
                      , mtl
+                     , text
+
                      , HUnit
                      , MissingH
                      , text
```
Comment (keremispirli) [inline sys/behave/behave.cabal]: Already on line 70

Comment (pjljvandelaar) [reply sys/behave/behave.cabal]: Two different items: test-suite (line 44) and library (line 15)

Comment (keremispirli) [reply sys/behave/behave.cabal]: No, same file line 70 and 74. Check the last line up there, under MissingH.

Comment (pjljvandelaar) [reply sys/behave/behave.cabal]: removed in deadcode pull request!

**Pull request merged**